### PR TITLE
Creating express server

### PR DESCRIPTION
**Pull request by jontkoh**
Hey guys so this is just the initial commit to dev with the bare bones of the frontend and backend. 

Next things are getting this to work on electron and creating a workable executable.

### CODE REVIEW

**Review by philipbaledia — APPROVED** [review]

File: client/src/App/pages/List.js [hunk 1]
```diff
@@ -1,51 +1,35 @@
...
-  // Fetch the list 
-  componentDidMount() {
-    fetch('http://localhost:8000/api/getList')
-    .then((res) => res.json())
-    .then(list => this.setState({ list }));
-  }
+function List() {
 
+  const [data] = useFetch(
+    'http://localhost:8000/api/getList'
```
Comment (DChan0319) [inline client/src/App/pages/List.js:12]: Put the api URL as a env variable.

File: client/src/App/pages/List.js [hunk 2]
```diff
@@ -1,51 +1,35 @@
...
+    <CenterContainer>
+      <div>
+      <Title>List from server{"\n"}</Title>
+        {data.map((item) => {
+          return (
+            <NextLine>
+              {item}
+            </NextLine>
+          )
+            {/* {console.log("from list.js - data: " + data)} */}
```
Comment (DChan0319) [inline client/src/App/pages/List.js:27]: is this comment still needed?

File: client/src/App/utils/hooks.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
+// Learned how to use the useEffect() for fetch() from:
+// https://medium.com/@cwlsn/how-to-fetch-data-with-react-hooks-in-a-minute-e0f9a15a44d6
+import { useState, useEffect } from 'react';
+
+function useFetch(url){
```
Comment (DChan0319) [inline client/src/App/utils/hooks.js:5]: add a space after ```(url) {```

File: client/src/index.html [hunk 4]
```diff
@@ -1,10 +1,10 @@
-<html lang="en">
+<html lang='en'>
```
Comment (DChan0319) [inline client/src/index.html:1]: you can leave your quotes as double quotes for html. It's more of the convention for html.

**Review by DChan0319 — COMMENTED** [review]
Looks like you fixed some things that I mentioned from last time. Small amount of comments that i've left. One that i didn't comment about is the css/styling about using ```px```.

**Pull request merged**